test/select_tests.cpp

```diff
@@ -348,7 +348,8 @@ TEST(SelectJoin) {
   ASSERT_STREQ(inner_join->left->name, "fact");
   ASSERT_EQ(inner_join->right->type, kTableName);
   ASSERT_STREQ(inner_join->right->name, "City")
```

Comment:
missing ;

Reply:
right. But it parses. Had a look into the microtest header and there, semicolons are already set (even superfluous ones: the basic asserts have semicolons already and the macros that build upon them add extra semicolons... 🥴). Is that something we want to touch? In the best case, this just means that we remove some semicolons in the microtest header.

Reply:
https://github.com/hyrise/sql-parser/blob/c2471248cef8cd33081e698e8ac65d691283dbd4/test/thirdparty/microtest/microtest.h#L24-L41

Reply:
Since we cannot touch the third party lib, I would care about our code and here missing semicolons bother me. :)

I don't mind if the compiled code ends up having a few unnecessary semicolons.

Reply:
Just adapted the header, can revert if you like. It helped spotting multiple places where assertions did not have semicolons at the end. IMO, if macros have semicolons internally, we should not add superfluous ones to make their calls more function-like.

Reply:
Love it!